ui/src/components/microsegmentation/AddSegmentation.js

```diff
@@ -1343,18 +1357,55 @@ class AddSegmentation extends React.Component {
                 {this.state.PESList.map((x, i) => {
                     return (
                         <div>
+                            <SectionDiv>
+                                <StyledInputLabel>Scope</StyledInputLabel>
+                                <CheckBoxSectionDiv>
+                                    <StyledCheckBox
+                                        checked={x.scopeall === 'true'}
+                                        name={'scopeallCheckBox' + i}
+                                        id={'scopeallCheckBox' + i}
+                                        key={'scopeallCheckBox' + i}
+                                        label='All'
+                                        onChange={(e) =>
+                                            this.handleInputChange(e, i)
+                                        }
+                                    />
+                                    <StyledCheckBox
+                                        checked={x.scopeonprem === 'true'}
+                                        disabled={x.scopeall === 'true'}
+                                        name={'scopeonpremCheckBox' + i}
+                                        id={'scopeonpremCheckBox' + i}
+                                        key={'scopeonpremCheckBox' + i}
+                                        label='On-Prem'
+                                        onChange={(e) =>
+                                            this.handleInputChange(e, i)
+                                        }
+                                    />
+                                    <StyledCheckBox
+                                        checked={x.scopeaws === 'true'}
+                                        disabled={x.scopeall === 'true'}
+                                        name={'scopeawsCheckBox' + i}
+                                        id={'scopeawsCheckBox' + i}
+                                        key={'scopeawsCheckBox' + i}
+                                        label='AWS'
+                                        onChange={(e) =>
+                                            this.handleInputChange(e, i)
+                                        }
+                                    />
+                                </CheckBoxSectionDiv>
+                            </SectionDiv>
```

Comment:
As discussed with @MartinTrojans , the policy enforcement state radio buttons depend on the user's scope input selection therefore they should come after the scope check boxes. 

**Updated Form Order**
<img width="1091" alt="image" src="https://github.com/AthenZ/athenz/assets/25251812/fc9a83af-8fb6-4f14-a1c9-cd52a58b2f71">

Reply:
The only file changed was `AddSegmentation.js`. All other files are changed from fix-lint